src/Semantic/CLI.hs

```diff
@@ -200,7 +201,7 @@ graphCommand = command "graph" (info graphArgumentsParser (progDesc "Compute a g
 shaReader :: ReadM Git.OID
 shaReader = eitherReader parseSha
   where parseSha arg = if length arg == 40 || arg == "HEAD"
-          then Right (Git.OID (T.pack arg))
+          then Right (Git.OID (B.pack arg))
```

Comment:
The thesis here is that this is safe (despite the `Char8` thing) because we’re talking about OIDs which are hex characters in ASCII, yes?

Do we have guardrails on that? Do we need any?

Reply:
They are always guaranteed to be in ASCII. I don’t think we need them, since I think this code path is going away?